### PR TITLE
Added http://acompas.audio to the samples page

### PR DESCRIPTION
**Pull request by oricordeau**

### CODE REVIEW

**Comment by oricordeau** [comment]
I signed the Contributor License Agreement !

File: samples/audio/samples.html [hunk 1]
```diff
@@ -503,6 +503,20 @@
 </tr> 
 
 
+<tr>
+<td style="width: 60px;">
+<div style="display: block; text-align: left;"><a href="http://acompas.audio" imageanchor="1"><img src="images/acompas.png" border="1"></a></div>
+</td>
+<td style="width: 3000px;"><a href="http://acompas.audio">
```
Comment (hoch) [inline samples/audio/samples.html]: Can you put `target="_blank"` in the `<a>` element?

Sorry for asking this, but eventually this whole page will be cleaned up at some point.

Comment (oricordeau) [inline samples/audio/samples.html]: Done !

**Comment by hoch** [comment]
Thanks for the submission!

**Comment by oricordeau** [comment]
You're welcome !

**Pull request merged**